src/components/ProgressBar.module.scss

```diff
@@ -0,0 +1,41 @@
+@use './../variables.scss';
+
+$initialBarWidth: 15px;
+
+.progressBar {
+  appearance: none;
+  height: 5px;
+  width: calc(100% - $initialBarWidth);
+  position: absolute;
+  top: 0;
+  left: $initialBarWidth;
+
+  // Add some initial width
```

Comment:
The initial value is not working for me in Safari 🤔

Reply:
Good catch, the ::after pseudo element was not working in Safari. I added a `<span>` on the DOM.